jtec-core/src/main/java/edu/tum/sse/jtec/instrumentation/testevent/TestEventInterceptorUtility.java

```diff
@@ -66,10 +69,20 @@ public static void executionStarted(final TestIdentifier testIdentifier, final D
         if (description != null) {
             if (testIdentifier.isTest()) {
                 testStarted(description);
+            } else if (testIdentifier.isContainer() && testIdentifier.getSource().isPresent()) {
```

Comment:
An inline comment for why the `getSource().isPresent()` is necessary would be nice to have.